### ISSUE TITLE
width: stretch not be prefixed when chrome >=46

### ISSUE THREAD

**Issue by butsalt** [postcss/autoprefixer#1071]
Sorry, I made a spell mistake in #1067. But the problem does exist. The prefix come back only until I change env to `Chrome >= 45`

env:
```
Chrome >= 46
```

input:
```
.a {
  width: stretch;
}
```

expect output: (chrome not support `width: stretch`, even chrome 46.0)
```
.a {
  width: -webkit-fill-available;
  width: stretch;
}
```

exact output:
```
.a {
  width: stretch;
}
```

**Comment by ai** [comment]
Can I Use shows that Chrome 46 doesn't need a prefix

**Comment by ai** [comment]
Fixed c4488fe

**Comment by ai** [comment]
Released in 8.6.4. Thanks for the report.
